core/domain/platform_parameter_domain_test.py

```diff
@@ -456,111 +472,231 @@ def test_evaluate_lte_version_filter_with_large_version_returns_false(self):
         self.assertFalse(filter_domain.evaluate(
             self._create_example_context(app_version='2.0.0')))
 
-    def test_evaluate_version_flavor_alpha_filter_equals_alpha_returns_true(
+    def test_evaluate_eq_version_flavor_filter_with_small_flavor_returns_false(
             self):
         filter_domain = (
             parameter_domain
             .PlatformParameterFilter.from_dict(
-                {'type': 'app_version_flavor', 'conditions': [('=', 'alpha')]}
+                {'type': 'app_version_flavor', 'conditions': [('=', 'test')]}
             )
         )
-        self.assertTrue(filter_domain.evaluate(
+        self.assertFalse(filter_domain.evaluate(
             self._create_example_context(app_version='1.10.0-abcdef-alpha')))
+        self.assertFalse(filter_domain.evaluate(
+            self._create_example_context(app_version='1.10.0-abcdef-beta')))
+
+    def test_evaluate_eq_version_flavor_filter_with_same_flavor_returns_true(
+            self):
+        filter_domain = (
+            parameter_domain
+            .PlatformParameterFilter.from_dict(
+                {'type': 'app_version_flavor', 'conditions': [('=', 'test')]}
+            )
+        )
+        self.assertTrue(filter_domain.evaluate(
+            self._create_example_context(app_version='1.10.0-abcdef-test')))
+
+    def test_evaluate_eq_version_flavor_filter_with_large_flavor_returns_false(
+            self):
+        filter_domain = (
+            parameter_domain
+            .PlatformParameterFilter.from_dict(
+                {'type': 'app_version_flavor', 'conditions': [('=', 'test')]}
+            )
+        )
+        self.assertFalse(filter_domain.evaluate(
+            self._create_example_context(app_version='1.10.0-abcdef-release')))
+
+    def test_evaluate_eq_version_flavor_filter_with_none_flavor_returns_false(
```

Comment:
Here & elsewhere: suggest referring to absent flavor as 'unspecified' since 'none' has a specific meaning in Python that's not applicable here.

Reply:
Done.